crypto/src/key/rschnorr/mod.rs

```diff
@@ -15,7 +15,9 @@
 
 mod internal;
 
+use crate::hash::{Blake2b32Stream, StreamHasher};
 use crate::random::{CryptoRng, Rng};
+use bitcoin::secp256k1;
```

Comment:
Why are we doing this implementation in the same file... rschnoor? why not start another separate submodule where we put the new secp256k1 stuff?

Please create a directory, call it secp256k1schnorr, and put the new stuff in it... no need to crowd this file... it's becoming confusing.